packages/aws-cdk/lib/notices.ts

```diff
@@ -108,8 +109,18 @@ export class WebsiteNoticeDataSource implements NoticeDataSource {
   fetch(): Promise<Notice[]> {
     const timeout = 3000;
     return new Promise((resolve, reject) => {
+      var req: ClientRequest;
```

Comment:
Please make this a `let`. Also, let's be a bit more honest about the type.

```suggestion
      let req: ClientRequest | undefined;
```